buildres/linux/postinst

```diff
@@ -19,10 +19,14 @@ set -e
 
 case "$1" in
     configure)
+        # Install the native-messaging host script for firefox/chrome/chromium
         install -D -m0755 /opt/jabref/lib/native-messaging-host/firefox/org.jabref.jabref.json /usr/lib/mozilla/native-messaging-hosts/org.jabref.jabref.json
         install -D -m0755 /opt/jabref/lib/native-messaging-host/chromium/org.jabref.jabref.json /etc/chromium/native-messaging-hosts/org.jabref.jabref.json
         install -D -m0755 /opt/jabref/lib/native-messaging-host/chromium/org.jabref.jabref.json /etc/opt/chrome/native-messaging-hosts/org.jabref.jabref.json
-        DESKTOP_COMMANDS_INSTALL
+        # Trigger an auto-install of the browser addon for chrome/chromium browsers
+        install -D -m0644 /opt/jabref/lib/native-messaging-host/chromium/bifehkofibaamoeaopjglfkddgkijdlh.json /opt/google/chrome/extensions/bifehkofibaamoeaopjglfkddgkijdlh.json
+        install -D -m0644 /opt/jabref/lib/native-messaging-host/chromium/bifehkofibaamoeaopjglfkddgkijdlh.json /usr/share/google-chrome/extensions/bifehkofibaamoeaopjglfkddgkijdlh.json
+	DESKTOP_COMMANDS_INSTALL
```

Comment:
Minor: indent is off here

Reply:
ah.. thanks. I guess the IDE changed my settings.